geohash.go

```diff
@@ -142,3 +152,45 @@ func (r Range) Round() float64 {
 	}
 	return roundDecimal(r.Mid(), dec)
 }
+
+// Return the neighbors of the supplied geohash. They are returned as a slice,
+// starting at the neighbor to the north, and going around clockwise with the
+// last being the neighbor to the northwest. See diagram below:
+//
+// 7  0  1
+// 6  x  2
+// 5  4  3
+
+func Neighbors(gh string) ([]string, error) {
```

Comment:
Ok, happy to make that change, it just seemed a common use case (and mine particularly) to iterate over these neighbors, so having a rigid struct seemed unnecessary, but I can make it a struct and leave it as an exercise to the suer to convert it to a slice